testsuite/Utils/math.cc

```diff
@@ -21,3 +21,28 @@ BOOST_AUTO_TEST_CASE(test_is_power_of_two)
     BOOST_CHECK(is_power_of_two(2048));
     BOOST_CHECK(!is_power_of_two(2049));
 }
+
+
+BOOST_AUTO_TEST_CASE(test_next_power_of_two)
+{
+    BOOST_CHECK_EQUAL(next_power_of_two(1), 1);
+
+    BOOST_CHECK_EQUAL(next_power_of_two(2), 2);
+
+    BOOST_CHECK_EQUAL(next_power_of_two(3), 4);
+    BOOST_CHECK_EQUAL(next_power_of_two(3), 4);
```

Comment:
Why check it twice?

Reply:
It cannot hurt :-) No, likely ```BOOST_CHECK_EQUAL(next_power_of_two(4), 4);``` was intended.